src/sagemaker/workflow/utilities.py

```diff
@@ -13,16 +13,20 @@
 """Utilities to support workflow."""
 from __future__ import absolute_import
 
+from pathlib import Path
 from typing import List, Sequence, Union
 import hashlib
 from urllib.parse import unquote, urlparse
+from _hashlib import HASH as Hash
```

Comment:
nit: coudn't find anything using this.

Reply:
It is used on lines 79, 97 and 115 for type hints